File: RxOperators/app/src/main/java/me/gurpreetsk/rxoperators/ui/filtering/FilteringOperatorsActivity.java
package me.gurpreetsk.rxoperators.ui.filtering;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import me.gurpreetsk.rxoperators.R;

public class FilteringOperatorsActivity extends AppCompatActivity {

  private static final String TAG = FilteringOperatorsActivity.class.getSimpleName();

  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_filtering_operators);
    setTitle(TAG);
  }
}

File: RxOperators/app/src/main/kotlin/me/gurpreetsk/rxoperators/ui/filtering/FilteringOperatorsActivity.kt
package me.gurpreetsk.rxoperators.ui.filtering

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import me.gurpreetsk.rxoperators.R

class FilteringOperatorsActivity : AppCompatActivity() {
    private val TAG = FilteringOperatorsActivity::class.java.simpleName

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_filtering_operators)
        title = TAG
    }
}
